docker-compose.yml

```
@@ -22,18 +22,13 @@ services:
       retries: 5
       start_period: 15s
-
-    #to enable replica set, requirement for enabling transactions
-    command: >
-      sh -c "
-        mongod --replSet rs0 --bind_ip_all --logpath /var/log/mongodb.log --logappend &
-        sleep 5 &&
-        mongosh --eval 'try { rs.initiate() } catch(e) { print(e) }' &&
-        tail -f /var/log/mongodb.log
-      "
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)